README.md

```diff
@@ -1 +1,215 @@
-# unicode-characters
+---
+datapackage:

```

Comment:
Not sure if we want to have datapackage metadata as frontmatter here. @solo11 could you please let me know what was the intention here?